coderd/provisionerdaemons.go

```diff
@@ -0,0 +1,79 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/coderd/database/db2sdk"
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/coderd/httpmw"
+	"github.com/coder/coder/v2/coderd/provisionerdserver"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+// @Summary Get provisioner daemons
+// @ID get-provisioner-daemons
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Enterprise
+// @Param organization path string true "Organization ID" format(uuid)
+// @Param tags query object false "Provisioner tags to filter by (JSON of the form {'tag1':'value1','tag2':'value2'})"
+// @Success 200 {array} codersdk.ProvisionerDaemonWithStatus
+// @Router /organizations/{organization}/provisionerdaemons [get]
+func (api *API) provisionerDaemons(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Is there a reason why endpoint changes and CLI go together? Alternatively, you could split the PR in half, and keep all golden files aside. Large PRs are typically a problem when somebody needs to revert them.

Reply:
Yes and no, for development I'm working on them in tandem, CLI requirements have and keep resulted in API changes, etc. Stacking could be a better approach though.

I don't think the golden files should be separate as that would make the feature hard to revert, but API and CLI can definitely be.

Reply:
It should be possible to have the DB stuff in its own PR and the CLI stuff then in a separate PR based off that. No need to have a really deep stack here IMO.

I'm fine with reviewing initially here "all-in-one".

Reply:
👍